README.md

```diff
@@ -1,2 +1,83 @@
-# wallet
-A budget handler application
+# Wallet Application 
+
+
+## Database deploy
```

Comment:
We're not deploying the database at all. The docker container is only used for local development, so maybe this wording is better.

```suggestion
## Running the database
```